deepchecks/nlp/text_data.py

```diff
@@ -447,3 +514,13 @@ def is_sampled(self, n_samples: t.Optional[int]):
         if n_samples is None:
             return False
         return self.n_samples > n_samples
+
+
+@contextlib.contextmanager
+def disable_deepchecks_logger():
+    """Disable deepchecks root logger."""
+    logger = get_logger()
+    logger_state = logger.disabled
+    logger.disabled = True  # Make sure we won't get the warning for setting class in the non multilabel case
```

Comment:
plz validate it when doing https://linear.app/deepchecks/issue/DEE-440/verify-support-in-token-classification-error-messages